src/script/calling/CallingRepository.ts

```diff
@@ -558,13 +558,25 @@ export class CallingRepository {
     );
   }
 
+  /**
+   * Will extract the conversation that is referred to by the calling event.
+   * It could happen that messages relative to a particular conversation was sent in a different conversation (most likely the self conversation).
+   * We need to find the correct conversation in order for AVS to know which call is concerned
+   */
+  private extractTargetedConversationId(event: CallingEvent): QualifiedId {
+    const {targetConversation, conversation, qualified_conversation} = event;
```

Comment:
I think its better to rename it to qualifiedConversation to maintain the naming convention. in the next line we are using 2 different convention otherwise

Reply:
Yeah, this thing with encoded message payloads is that we have those 2 convention. 
But, TBH, it feels like a waste of resources to rename at this point since it's super located and that this variable is renamed to a camel cased variable on the next line. 
WDYT?